app/src/main/res/values/strings.xml

```diff
@@ -128,7 +128,7 @@
     <string name="history_metadata_header_2">Recently visited</string>
     <!-- Text for the menu button to remove a grouped highlight from the user's browsing history
          in the Recently visited section -->
-    <string name="recently_visited_delete_from_history">Delete from history</string>
```

Comment:
Let not remove strings directly as it could cause crashes, in older versions instead use the deletion process https://github.com/mozilla-mobile/fenix/wiki/Remove-strings

Reply:
I'd say normally, yes, but this just landed yesterday and can't be in any other version yet. So seems fine?

Reply:
👍🏽